app/src/main/kotlin/com/intive/picover/parties/repository/MockedPartiesRepository.kt

```diff
@@ -0,0 +1,21 @@
+package com.intive.picover.parties.repository
+
+import com.intive.picover.parties.model.Party
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class MockedPartiesRepository @Inject constructor() {
```

Comment:
There is no need to keep "Mocked" in the name. Repositories are intended to be used as a data source, without explicitly saying by the name, what is the origin of the data, so there should be no names like `Mocked`/`Cached`/`Stored`/`Api` etc.